diff --git a/mobile/authority/local-story/save.ts b/mobile/authority/local-story/save.ts
index 651bbe7a..493d6e5b 100644
--- a/mobile/authority/local-story/save.ts
+++ b/mobile/authority/local-story/save.ts
@@ -95,7 +95,7 @@ function reconciled(s: Story, r: Receipt, checkpoint?: Checkpoint) {
           'unexpected elapsed checkpoint; recovery required',
         );
     }
-    return got;
+    return undefined;
   });
 }
 
