diff --git a/mobile/app/metro.config.js b/mobile/app/metro.config.js
index a137172..1478bdd 100644
--- a/mobile/app/metro.config.js
+++ b/mobile/app/metro.config.js
@@ -6,5 +6,7 @@ const config = getDefaultConfig(__dirname);
 config.watchFolders = [
   path.resolve(__dirname, '..'),
   path.resolve(__dirname, '../../kernel/ts/src'),
+  path.resolve(__dirname, '../../kernel/ts/test'),
+  path.resolve(__dirname, '../../protocol/fixtures'),
 ];
 module.exports = config;
